src/DebugBar/DataCollector/ExceptionsCollector.php

```diff
@@ -29,6 +31,9 @@ public function addException(Exception $e)
     {
         $this->exceptions[] = $e;
         if ($this->chainExceptions && $previous = $e->getPrevious()) {
+            if ($previous instanceof Error) {
```

Comment:
And while https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Debug/Exception/FatalThrowableError.php accepts Throwable, it seems to be explicitly aimed at the Error class.